README.md

```diff
@@ -214,6 +214,7 @@ If you're not inclined to make PRs you can tweet me at ```@ripienaar```
    * http://www.acunote.com/ - Free project management and SCRUM software for up to 5 team members.
    * http://gliffy.com/ - Online diagrams: flowchart, UML, wireframe... Also Plugins for Jira & Confluence. 5 diagrams and 2 MB free.
    * https://cacoo.com/ - Online diagrams in real time: flowchart, UML, network. Free max. 15 users/diagram, 25 sheets.
+   * http://backlogtool.com/ - Project Management with integrated net tools. Free 1 project, 10 users, no Gantt.
```

Comment:
I don't see the point of this service in its free form, its just a git diff tool and file sharing, no point at all and doesnt let you experience any of the features its really used for. Just not useful for this list sorry